### PR TITLE
MAUT-8990 : Upgrade to PHP8

### PR DESCRIPTION
**Pull request by tejasync**
Support PHP8 upgrade

### CODE REVIEW

**Review by rohitp19 — DISMISSED** [review]
Approving as team already reviewed it.

**Review by dadarya0 — DISMISSED** [review]

File: EventListener/TokenSubscriber.php [hunk 1]
```diff
@@ -17,6 +17,7 @@
 use MauticPlugin\CustomObjectsBundle\DTO\TableConfig;
 use MauticPlugin\CustomObjectsBundle\DTO\Token;
 use MauticPlugin\CustomObjectsBundle\Entity\CustomField;
+use MauticPlugin\CustomObjectsBundle\Entity\CustomFieldValueInt;
```
Comment (dadarya0) [inline EventListener/TokenSubscriber.php]:
```suggestion
```

**Review by dadarya0 — CHANGES_REQUESTED** [review]
found 1 minor change.

**Review by dadarya0 — APPROVED** [review]

**Review by dadarya0 — APPROVED** [review]

**Review by rohitp19 — APPROVED** [review]
LGTM.

**Review by dadarya0 — APPROVED** [review]

**Review by rohitp19 — APPROVED** [review]

**Pull request merged**